package-res/resources/web/pentaho/data/Filter.js

```diff
@@ -32,6 +46,66 @@ define([
     }
   });
 
+  /**
+   * @name Filter
+   * @memberOf pentaho.data
+   * @class
+   * @amd pentaho/data/Filter
+   *
+   * @classdesc The `Filter` class provides factory methods for building the following filters:
+   *
```

Comment:
This is no longer true. It provides a  single `create` factory function and a bunch of classes.